web/src/engine/websites/SpoilerPlus.ts

```diff
@@ -0,0 +1,189 @@
+import { Tags } from '../Tags';
+import icon from './SpoilerPlus.webp';
+import {type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchWindowScript } from '../platform/FetchProvider';
+import type { Priority } from '../taskpool/DeferredTask';
+import DeScramble from '../transformers/ImageDescrambler';
+
+const pageScript = `
+    new Promise( resolve => {
```

Comment:
If this is custom code use more expressive variable names, `a`, `c`, `i`  and `s` are quite difficult to understand.
May add a comment with optional link to script in case this code was directly copied from the website.

Reply:
Its from the website, but code was more than heavily obfuscated so it went under heavy cleaning (which was already a pain cause they reuse identifiers everywhere). I am afraid a link woulnt help 🤷 .